Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ however, it has to be formatted properly to pass verification tests.
 - Added Dual Sense Edge controller to be mapped to the same layout as the Dual Sense controller
 - Added drag and drop support in the Input Action Asset Editor for Action Maps, Actions and Bindings.
 - UI Toolkit input action editor now supports showing the derived bindings.
+- Added right-click (context) menu support for empty areas below the Action Maps/Actions.
```

Comment:
Since Action Maps / Actions UI exist in many places, editors, Inspector property drawers of various sorts etc I would recommend being more specific about the context this applies to: Project Settings Input Action editor and Asset Input Action Editor.